src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,41 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final int MAX_FIGURE_TYPES = 5;
+    private final ColorSupplier colorSupplier;
+    private final Random random;
+
+    public FigureSupplier() {
+        this.colorSupplier = new ColorSupplier();
+        this.random = new Random();
+    }
+
+    private double getRandomSide() {
+        return random.nextDouble() * 100 + 1;
```

Comment:
use constants instead of numbers
[this article](https://mate-academy.github.io/style-guides/java/java.html#s5.2.4-constant-names)